### PR TITLE
Dronov Ilya 2537

### PR DESCRIPTION
**Pull request by DronovIlya**

### CODE REVIEW

**Comment by thevery** [comment]
refresh button is almost invisible

**Comment by thevery** [comment]
UX issues: tabs+refresh button takes too much space (should be moved to action bar), action bar is useless on the second screen (should be removed)

**Comment by thevery** [comment]
previews are scaled, not cropped

**Comment by thevery** [comment]
second screen image quality is bad

**Comment by thevery** [comment]
refresh should be non-blocking (though progress is better than nothing)

**Comment by thevery** [comment]
app is ok (20-5=15), code review pending

**Comment by DronovIlya** [comment]
Fixed some bug. Please, check

File: res/layout/main.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<LinearLayout
```
Comment (thevery) [inline res/layout/main.xml:1]: useless layout

File: src/com/android/example/dronov/ImageActivity.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * To change this template use File | Settings | File Templates.
+ */
+public class ImageActivity extends Activity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);    //To change body of overridden methods use File | Settings | File Templates.
+        setContentView(R.layout.image_view);
+
+        Intent intent = getIntent();
+        int index = intent.getIntExtra("bitmap", 0);
```
Comment (thevery) [inline src/com/android/example/dronov/ImageActivity.java:24]: `"bitmap"` should be const

File: src/com/android/example/dronov/MyActivity.java [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+        DisplayMetrics dimension = new DisplayMetrics();
+        getWindowManager().getDefaultDisplay().getMetrics(dimension);
+        width = dimension.widthPixels;
+        height = dimension.heightPixels;
+
+
+
+        layout = (GridView) findViewById(R.id.grid_view);
+        if (isPortraitOrientaion) {
```
Comment (thevery) [inline src/com/android/example/dronov/MyActivity.java:67]: use `layout` and `layout-land` resource folders instead

File: src/com/android/example/dronov/MyActivity.java [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        }
+        layout.setStretchMode(GridView.STRETCH_SPACING_UNIFORM);
+
+
+        database = new YandexPhotos(this);
+        database.open();
+        bitmapList = database.getAllPicturesData();
+        getSmallBitmap();
+        if (bitmapList.isEmpty()) {
+            Toast toast = Toast.makeText(getApplicationContext(),
```
Comment (thevery) [inline src/com/android/example/dronov/MyActivity.java:86]: strings.xml

**Comment by thevery** [comment]
20-5-3=12

**Pull request closed without merging**